tests/test_markov_model.py

```diff
@@ -612,3 +614,31 @@ def test_get_mean_waiting_time_from_closed_form_markov():
     assert round(mean_waiting_time, number_of_digits_to_round) == round(
         1.1051493390764142, number_of_digits_to_round
     )
+
+
+def test_generate_code_for_tikz_figure():
```

Comment:
This test looks good, nicer to have them as four separate functions though, so that when the tests fail you might get more immediate information.

Reply:
Sure thing. I will separate them now.